index.html

```diff
@@ -258,5 +258,70 @@ <h2>I’am a Software Developer</h2>
         </div>
       </div>
     </section>
+
+    <!--Working On Form-->
+
+    <div class="form-section">
+      <div class="form-headline">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </div>
+      <form
+        class="form-sementic"
+        action="https://formspree.io/f/mayvlpor"
+        method="POST"
+      >
+        <div>
+          <input
+            type="text"
+            name="user_name"
+            placeholder="Full name"
+            class="user-name"
+            minlength="5"
+            maxlength="30"
+            required
+          />
+        </div>
+        <div>
+          <input
+            type="email"
+            name="user_email"
+            placeholder="Email address"
+            class="user-email"
+            required
+          />
+        </div>
+        <div>
+          <textarea
+            name="user_Message"
+            class="msg-area"
+            placeholder="Enter text here"
+            maxlength="500"
+          ></textarea>
```

Comment:
- [ ] Please be noticed that the textarea must have a required attribute to stop submitting the form without filling it.